src/base/datetime/calendar/body/index.ts

```diff
@@ -0,0 +1,294 @@
+        ${this._getHeaderItemsTemplate()}<!--
+        -->${this._getDateItemsTemplate()}
+      </table>
+    `;
+  }
+
+  private _getHeaderItemsTemplate() {
+    return html`
+      <thead>
+        ${this._locale.WEEK_DAYS.map(wday => {
+          return html`
+            <th
+              class="kuc-base-datetime-calendar-body__header"
+              role="columnheader"
+              abbr="${wday.abbr}"
+            >
+              ${wday.text}
+            </th>
+          `;
+        })}
+      </thead>
+    `;
+  }
+
+  private _getDateItemsTemplate() {
+    const displayingDates = getDisplayingDates(this.year, this.month);
+    const today = this._getDateString();
+    return html`
+      <tbody>
+        ${displayingDates.map(weeks => {
+          return html`
+            <tr>
+              ${weeks.map(weekDate => {
+                const dateParts = weekDate.split("-");
+                return html`
+                  <td role="gridcell">
+                    <button
+                      aria-selected="${this.value === weekDate}"
+                      tabindex="${weekDate === today || this.value === weekDate
+                        ? "0"
+                        : "-1"}"
+                      class="kuc-base-datetime-calendar-body__date${this._getDateClass(
+                        dateParts
+                      )}"
+                      data-date="${weekDate}"
+                      @click=${this._handleClickDateCalendarBody}
+                      @keydown="${this._handleKeyDownDateCalendarBody}"
+                    >
+                      ${dateParts[2] || ""}
+                    </button>
+                  </td>
+                `;
+              })}
+            </tr>
+          `;
+        })}
+      </tbody>
+    `;
+  }
+
+  private _handleClickDateCalendarBody(event: MouseEvent | KeyboardEvent) {
+    event.preventDefault();
+    event.stopPropagation();
+
+    const itemEl = event.target as HTMLButtonElement;
+    itemEl.setAttribute("aria-selected", "true");
+
+    const value = itemEl.getAttribute("data-date") || "";
+    this._handleDispatchEvent(value);
+    this._close();
+  }
+
+  private _handleKeyDownDateCalendarBody(event: KeyboardEvent) {
+    let flag = false;
+    switch (event.key) {
+      case "Up":
+      case "ArrowUp": {
+        flag = true;
+        this._moveToDate(-7);
+        break;
+      }
+      case "Down":
+      case "ArrowDown": {
+        flag = true;
+        this._moveToDate(7);
+        break;
+      }
+      case "Left":
+      case "ArrowLeft": {
+        flag = true;
+        this._moveToDate(-1);
+        break;
+      }
+      case "Right":
+      case "ArrowRight": {
+        flag = true;
+        this._moveToDate(1);
+        break;
+      }
+      case "Enter": {
+        flag = true;
+        const value = this._getValueSelected();
+        this._handleDispatchEvent(value);
+        this._close();
```

Comment:
@nmanhit
It will make the body disapear when clicking.
Please remove it.

It will show/hide following Calendar (parent) part.